_posts/2020-07-29-Kamen-Kanev-CV.md

```diff
@@ -4,101 +4,119 @@ title: Kamen Kanev
 comment: Default CV
 ---
 # Kamen Kanev
-Entrepreneur and Engineer
+Senior Software Engineer and Entrepreneur
```

Comment:
I'd go without `Senior` here because having `Entrepreneur` makes it feel more like that line is answering the question of "what are you working" and "Senior" is more like a title.